samples/scales/time/financial.html

```diff
@@ -97,7 +97,6 @@
 
 		var color = Chart.helpers.color;
 		var cfg = {
-			type: 'bar',
```

Comment:
is this intended?

Reply:
Yes, it was just here as a workaround, but this change makes it no longer necessary. If you look at https://www.chartjs.org/samples/latest/scales/time/financial.html it starts out as a line chart, but there's a dropdown that allows you to change the chart type. That wasn't working correctly before, because it required scale options and not just controller options. This fixes that though